app/src/main/java/com/github/se/signify/model/stats/SaveStatsToFireStore.kt

```diff
@@ -26,7 +26,8 @@ fun saveStatsToFirestore() {
             "dailyQuest" to 0,
             "weeklyQuest" to 0,
             "completedChallenge" to 0,
-            "createdChallenge" to 0)
+            "createdChallenge" to 0,
+            "wonChallenge" to 0)
```

Comment:
Perhaps, these could be split into 3 categories of stats, in separate maps. What do you think?

Reply:
I agree, but when I need to test I don't know how to access all the element properly for mock.